src/librustc_middle/ty/context.rs

```diff
@@ -182,7 +182,7 @@ pub struct CommonLifetimes<'tcx> {
 }
 
 pub struct CommonConsts<'tcx> {
-    pub err: &'tcx Const<'tcx>,
+    pub unit: &'tcx Const<'tcx>,
```

Comment:
Where is this used?

Reply:
I didn't want to add a `PhantomData`, so I left this for https://github.com/rust-lang/rust/pull/70891.

Reply:
Ah ok, sounds good.